docs/bucket/notifications/README.md

```diff
@@ -464,6 +467,10 @@ When the _access_ format is used, Minio appends events to a table. It creates ro
 
 The steps below show how to use this notification target in `namespace` format. The other format is very similar and is omitted for brevity.
 
+### Step 0: Ensure minimum requirements are met
```

Comment:
Hate to nitpick. Please use step 1, step 2  instead of 0, 1. Just to be consistent with other docs.